src/status_im/contexts/settings/wallet/events.cljs

```diff
@@ -82,3 +88,81 @@
            :on-error   #(log/info "failed to remove keypair " %)}]]]})
 
 (rf/reg-event-fx :wallet/remove-keypair remove-keypair)
+
+(defn make-keypairs-accounts-fully-operable
+  [{:keys [db]} [key-uids-to-update]]
+  (let [key-uids-set (set key-uids-to-update)
+        keypair-name (data-store/extract-keypair-name db key-uids-set)]
+    {:db (-> db
+             (update-in [:wallet :accounts] #(data-store/make-accounts-fully-operable % key-uids-set))
+             (update-in [:wallet :keypairs] #(data-store/make-keypairs-fully-operable % key-uids-set)))
+     :fx [[:dispatch
+           [:toasts/upsert
+            {:type  :positive
+             :theme :dark
+             :text  (if (= (count key-uids-to-update) 1)
+                      (i18n/label :t/key-pair-imported-successfully {:name keypair-name})
+                      (i18n/label :t/key-pairs-successfully-imported
+                                  {:count (count key-uids-to-update)}))}]]]}))
+
+(rf/reg-event-fx :wallet/make-keypairs-accounts-fully-operable make-keypairs-accounts-fully-operable)
```

Comment:
Nice! Definitely will be using this in my PR #20181 
Also this function might need to be updated depending on when https://github.com/status-im/status-mobile/pull/20160 merges.